📝(project) improve README (#489)
openfun/ralph: README.md

<p align="center">

>   <a href="https://openfun.github.io/ralph"><img src="https://raw.githubusercontent.com/openfun/logos/main/ralph/ralph-color-dark.png" alt="Ralph" width="400"></a>

Reviewer: ```suggestion
  <a href="https://openfun.github.io/ralph"><img src="https://raw.githubusercontent.com/openfun/logos/main/ralph/ralph-color-dark.png" alt="Ralph logo" width="400"></a>
```

</p>

<p align="center">

>     <em>Ralph, the ultimate Learning Record Store (and more!) for your learning analytics</em>

Reviewer 2: ```suggestion
    <em>⚙️ The ultimate toolbox for your learning analytics (expect some xAPI ❤️)</em>
```

Reviewer: ```suggestion
    <em>Ralph, the ultimate toolbox to run and manage your Learning Record Store</em>
```
? 

I like the idea that the LRS is mentionned in the title as it is a essential component of the library. Toolbox is truer as it encompases all functionalities, but is rather vague. TBH, the first time I saw the page, I did not get much out of this description.

Author: @Leobouloc I agree! I think it would be nice to emphasize on Ralph being an LRS in the "slogan", even if we're omitting the CLI and library abilities.
The CLI and library aspects of Ralph are detailed right after, under the 3 bullet points anyway.

Reviewer: Fine with me :)

</p>

<p align="center">
<a href="https://circleci.com/gh/openfun/ralph/tree/master">
    <img src="https://img.shields.io/circleci/build/gh/openfun/ralph/master?label=Tests&logo=circleci" alt="CircleCI tests">
</a>
<a href="https://pypi.org/project/ralph-malph">
    <img src="https://img.shields.io/pypi/v/ralph-malph?label=PyPi+package" alt="PyPi version">
</a>
<a href="https://pypi.org/project/ralph-malph">
    <img src="https://img.shields.io/pypi/pyversions/ralph-malph" alt="Python versions supported">
</a>

> <a href="https://discord.gg/vYx6YWxJCS">

Reviewer 2: ```suggestion
<a href="https://hub.docker.com/r/fundocker/ralph/tags">
    <img src="https://img.shields.io/docker/v/fundocker/ralph/latest?label=Docker+image" alt="DockerHub">
</a>
<a href="https://discord.gg/vYx6YWxJCS">
```
